katdal/datasources.py

```diff
@@ -191,9 +191,15 @@ class ChunkStoreVisFlagsWeights(VisFlagsWeights):
         Correlation products. If given, the weights for baseline (inp1, inp2)
         will be divided by the square root of the product of the corresponding
         autocorrelations vis[inp1,inp1] and vis[inp2,inp2].
+
+    Attributes
+    ----------
+    prefix : string
+        Prefix of correlator_data / visibility array (i.e. the S3 bucket name)
```

Comment:
I'd feel happier if we call this something like `correlator_data_prefix`, so that it's explicit that you shouldn't depend on using it for the other arrays. Otherwise the unwary might start using it like that, which would then be a source of future bugs e.g. code might accidentally use the L0 instead of the L1 flags.